src/compiler/diagnosticMessages.json

```diff
@@ -1517,6 +1517,10 @@
         "category": "Error",
         "code": 4089
     },
+    "Cannot initialize outer scope variable '{0}' when having block-scoped variable with the same name.": {

```

Comment:
Should be:
"Cannot initialize function scoped variable '{0}' in the same scope as block scoped declaration '{1}'."
or you can use "outer" instead of "function scoped"